### PR TITLE
Multi-region `aws_codedeploy`

### PR DESCRIPTION
**Pull request by aguillenv**
Part of #1260

### CODE REVIEW

File: test/modules/aws-codedeploy-integration.ts [hunk 1]
```diff
@@ -372,6 +373,68 @@ describe('AwsCodedeploy Integration Testing', () => {
...
 });
 
+describe('Move deployments to another region', () => {
+  it('should fail moving just the deployment group', done =>
+    void query(`
+      UPDATE codedeploy_deployment_group
+      SET region = '${nonDefaultRegion}'
+      WHERE name = '${deploymentGroupName}';
+  `)((e?: any) => {
+      console.log({ e });
```
Comment (dfellis) [inline test/modules/aws-codedeploy-integration.ts:383]: Should these `console.log` lines be removed?

Comment (aguillenv) [reply test/modules/aws-codedeploy-integration.ts:383]: Oh, yes I can remove it, I just copied it from another test

Comment (aguillenv) [inline test/modules/aws-codedeploy-integration.ts:383]:
```suggestion
```

**Review by dfellis — APPROVED** [review]

File: src/modules/0.0.22/aws_codedeploy/mappers/deployment.ts [hunk 2]
```diff
@@ -35,13 +34,19 @@ export class CodedeployDeploymentMapper extends MapperBase<CodedeployDeployment>
...
       return undefined;
-    out.application = await this.module.application.cloud.read(ctx, deployment.applicationName);
-    out.deploymentGroup = await this.module.deploymentGroup.cloud.read(ctx, deployment.deploymentGroupName);
+    out.application = await this.module.application.cloud.read(
+      ctx,
+      `${deployment.applicationName}|${region}`,
+    );
+    out.deploymentGroup = await this.module.deploymentGroup.cloud.read(
+      ctx,
+      `${deployment.deploymentGroupName}|${deployment.applicationName}|${region}`,
```
Comment (yrobla) [inline src/modules/0.0.22/aws_codedeploy/mappers/deployment.ts:48]: why not use entityIds there?

**Review by yrobla — APPROVED** [review]

**Pull request merged**